unabridged-works-volume-1/original/04-scattered-sheep.adoc

```diff
@@ -488,12 +495,21 @@ he being (by the strict observation of these) kept
 out of the fear of the danger of his condition,
 which otherwise perhaps he might be made sensible of.
 
+====
+
+[.discourse-part]
 Objection.
 But I may be deceived in hearkening to a light within;
 for while I think that I therein hearken to the light of the Spirit,
 it may prove but the light of a natural conscience.
 
+[.discourse-part]
 Answer.
```

Comment:
So this is on its own line because it's followed by a numbered group....dunno if that's ok?

Reply:
I think that's OK. 👍